src/main/java/donkeychat/Storage.java

```diff
@@ -0,0 +1,86 @@
+package donkeychat;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
```

Comment:
Maybe you can consider separating this import statement from the rest? 

Such as
```java
import java.io.File;
import java.io.FileNotFoundException;
import java.io.FileWriter;
import java.io.IOException;

import java.util.Scanner;
```